packages/regl-worldview/src/Worldview.js

```diff
@@ -65,6 +65,7 @@ export type BaseProps = {|
 
   // Context attributes passed into canvas.getContext.
   contextAttributes?: ?{ [string]: any },
+  offscreenCanvas?: HTMLCanvasElement,
```

Comment:
Any thoughts on lining up the name and the type? IMO it'd be nice to either call it `canvas` or have the type be either `OffscreenCanvas` or `HTMLCanvasElement | OffscreenCanvas`. But I don't mind this too much either.